### PR TITLE
Version-2.3.0

### PR DESCRIPTION
**Pull request by AnYiEE**

### CODE REVIEW

File: .nvmrc
Comment (diskdance) [inline .nvmrc:1]: Please don't delete this file as it tells the Node version of this project.

File: src/spacing.ts [hunk 1]
```diff
@@ -52,13 +52,13 @@ const SELECTOR = SELECTOR_ALLOWED
   .map((allowed) => `${allowed}:not(${SELECTOR_BLOCKED
     .flatMap((blocked) =>
       // Not include itself if it is a tag selector
-      blocked[0].match(/[a-z]/i) ? `${blocked} *` : [blocked, `${blocked} *`]
+      (blocked[0] as string).match(/[a-z]/i) ? `${blocked} *` : [blocked, `${blocked} *`],
```
Comment (diskdance) [inline src/spacing.ts:55]: Is this required? I didn't check it locally but from my assumption it should be `string`.

Comment (AnYiEE) [reply src/spacing.ts:55]: TypeScript infers that `blocked[0]` may be undefined, but I believe it is always a string. This is also to resolve the error of the `@typescript-eslint/recommended-type-checked` rule.

File: .nvmrc
Comment (AnYiEE) [reply .nvmrc:1]: Since the project does not impose a specific version of Node.js, this file has little significance.

Comment (AnYiEE) [reply .nvmrc:1]: I will keep it.

**Pull request merged**